Unable to compile using R17A

I tried compiling Elixir using [the R17A preview branch](https://github.com/erlang/otp/tree/egil/maps/eep-implementation), but I get an error when I try `make install`:

```
tojans@ubuntu:/tmp/elixir$ make install 
==> elixir (compile)
ERROR: OTP release R17A does not match required regex (R16B).*
ERROR: compile failed while processing /tmp/elixir/lib/elixir: rebar_abort
make: *** [erlang] Error 1
```

This is probably a really small fix...

--- a/lib/elixir/rebar.config
+++ b/lib/elixir/rebar.config
@@ -24,4 +24,4 @@
             {verbose, false}
            ]}.
 
-{require_otp_vsn,"(R16B).*"}.
+{require_otp_vsn,"(R16B|R17A).*"}.

